Source/WebCore/platform/network/ResourceResponseBase.h

```diff
@@ -267,6 +273,9 @@ class ResourceResponseBase {
     mutable UsedLegacyTLS m_usedLegacyTLS : bitWidthOfUsedLegacyTLS { UsedLegacyTLS::No };
     mutable WasPrivateRelayed m_wasPrivateRelayed : bitWidthOfWasPrivateRelayed { WasPrivateRelayed::No };
     String m_proxyName;
+#if ENABLE(LOCAL_NETWORK_ACCESS)
+    std::optional<IPAddressSpace> m_ipAddressSpace;
```

Comment:
Will this commonly be nullopt?  If so, we might want to use a unique_ptr to reduce sizeof(ResourceResponse)